api/server.py

```diff
@@ -1210,31 +1247,17 @@ def start_api_server():
 
     write_to_api_console(
         " * API access key: {0}\n".format(
-            api_access_key
-            if not api_access_without_key
-            else "NOT REQUIRED!"
+            api_access_key if not api_access_without_key else "NOT REQUIRED!"
         )
     )
 
     app.config["OWASP_HONEYPOT_CONFIG"] = {
-        "api_access_key":
-            api_access_key,
-
-        "api_client_white_list":
-            my_api_configuration["api_client_white_list"]["enabled"],
-
-        "api_client_white_list_ips":
-            my_api_configuration["api_client_white_list"]["ips"],
-
-        "api_access_log":
-            my_api_configuration["api_access_log"]["enabled"],
-
-        "api_access_log_filename":
-            my_api_configuration["api_access_log"]["filename"],
-
-        "api_access_without_key":
-            api_access_without_key,
-
+        "api_access_key": api_access_key,
+        "api_client_white_list": my_api_configuration["api_client_white_list"]["enabled"],
+        "api_client_white_list_ips": my_api_configuration["api_client_white_list"]["ips"],
```

Comment:
this is breaking the lint ,line too long (113 > 80 characters)

Reply:
I reconfigure flake8 to 120 chars now.